Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1

```diff
@@ -111,20 +113,16 @@ function GetRelativeFilePath($component)
 }
 
 function GetSonarReportProcessedFilePath
-{
-    param([string][ValidateNotNullOrEmpty()]$agentBuildDirectory)
```

Comment:
Many of the functions here take a single param - the build agent directory. This is unnecessary because the build agent dir is exposed through an env variable.